### ISSUE TITLE
Albums: Automatic suggestion of artists based on songs

### ISSUE THREAD

**Issue by andreoda** [VocaDB/vocadb#1613]
I noticed that users, especially new ones, forget to add vocalists or other artists like producers to album entries even when songs/tracks were already added to the album entry.
Especially on compilation albums this also happens to veteran users due to the amount of different artists.

A way to make artist adding on album entries easier is to have a suggestion system there that will show artists that participated in the songs that are already added to the album entry.
On the artist tab, users could just click on the suggested artists to add them.
This function should work in real-time, meaning that the artists should be suggested in the same session in which the user added the tracks to the entry so that the album entry does not need to be closed and opened again for it to fetch the suggestions.
(I hope I could explain well what I mean)

In short: An artist suggestion for album entries

- The system should suggest vocalists, producers, lyricists etc.
- The system should not suggest animators and illustrators
